Core/PARStore.m

```diff
@@ -1382,7 +1384,41 @@ - (BOOL)writeBlobFromPath:(NSString *)sourcePath toPath:(NSString *)targetSubpat
     return YES;
 }
 
+- (BOOL)writeTombstoneAtPath:(NSString *)tombstonePath forFileAtPath:(NSString *)filePath error:(NSError **)error
```

Comment:
Just to be clear, although I suggest using "registeredDeletion" in the API, the private methods can stick with tombstone.